src/main.py

```diff
@@ -88,6 +88,11 @@ def write_report(tests, drive, output_path):
 
     r.close()
 
+def restore_drive(config):
+    drive = config['drive']['name']
+    tree = n_utils.generate_resource_tree()
```

Comment:
Do you need a call to sedutil to do a reset?

Reply:
Thanks, I will add a reset.